ui/core/src/ui-core/style/itwinui-overrides.scss

```diff
@@ -0,0 +1,55 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+@import "../icons/variables";
+@import "../inputs/variables";
+@import "./themecolors";
+@import "./variables";
+@import "~@itwin/itwinui-css/scss/style/space";
+
+// Override some iTwinUI-css padding and minimums for iTwin.js applications
+.iui-input {
+  min-height: $uicore-component-height-normal !important;
+  padding: $uicore-iui-input-padding !important;
+}
+
+.iui-select {
+  >.iui-select-button {
+    min-height: $uicore-component-height-normal !important;
+    padding-top: 2px !important;
+    padding-bottom: 2px !important;
+    padding-right: 24px !important;
+  }
+
+  >.iui-select-button::after {
+    right: 6px !important;
+  }
+}
+
+.iui-toggle-switch {
+  padding: 1px 0 !important;
+}
+
+.iui-expandable-block {
+  > .iui-header {
+    padding: $iui-xs $iui-xs !important;
+
+    > .iui-icon {
+      width: $uicore-icons-x-small !important;
+      height: $uicore-icons-x-small !important;
+      margin-top: $iui-xs !important;
+    }
+  }
+

```

Comment:
The chevron looks very misaligned:
![image](https://user-images.githubusercontent.com/70327485/127645572-2b93e21d-afa5-4264-85d6-83355ee2451e.png)